building/glass/branches-schema.yaml

```diff
@@ -38,6 +38,8 @@ properties:
           pattern: "^[a-fA-F0-9]+$"
         upload-target:
           type: string
+        ci:
+          type: boolean
```

Comment:
I'm concerned about intentional divergences between normal development/production and CI. This could easily lead to situations where CI approves something, but it's broken. Do we necessarily need this kind of divergence?